### ISSUE TITLE
Container creation

### ISSUE THREAD

**Issue by olivierlambert** [xenserver/xscontainer#5]
Hi there!

We [started to use this plugin in XO](https://xen-orchestra.com/blog/xenserver-xen-orchestra-and-docker/) for displaying containers and offer life cycle actions on them.

Our next step is the VM "docker management ready" creation, and the step after that will be container creation.

Question: is it planned to create containers from this host plugin?

**Comment by robertbreker** [comment]
Yes, I'd definitely like to see a XenServer API for the creation of containers.
The challenge will be that container creation comes with a range of dependencies such as image management.
Before approaching this, I think we ought to put more thought into the API basics. It is questions, such as whether the container API should be implemented in a plugin going forward or directly in XAPI.

**Comment by olivierlambert** [comment]
Copy that. If there is any chance to follow the discussion about that somewhere, I'm in!
